### ISSUE TITLE
Where are the loss functions for ITC, ITM, and ITG in the first stage of training in blip2？

### ISSUE THREAD

**Issue by nuocheng** [salesforce/LAVIS#653]
Hello, I have carefully read articles about blip2, which has sparked a great interest in me. Also looked at the relevant core code of q transformer in blip2. I have a question and I need your help.
I found the use of cross entropy loss in the experimental section of the blip2 article, but I did not find the loss section in the code.
![image](https://github.com/salesforce/LAVIS/assets/31612743/d43b4a98-3a02-498a-91ce-0e514bc4b83d)
